README.md

```diff
@@ -6,6 +6,51 @@ Nevertheless, hopefully the guide can serve as training material to iOS app deve
 
 > __Just like any software, this guide will rot unless we update it. We encourage everyone to help us on that, just open an issue or send a pull request!__
 
+## Jaibroken Test
+
+Checking whether a device is jailbroken or not . Attacker can run tools like Cycript, GDB, Snoop-it etc to perform runtime analysis and steal sensitive data from within your application. This will help for extra layer of security for your application, you should not allow your application to be run on a jailbroken device.
```

Comment:
Checking whether a device is jailbroken or not …

I think there is something missing here in the sentence?

"Checking whether a device is jailbroken or not can be helpful to make certain in-app security decisions"

Also remove the last sentence.